jib-core/src/test/java/com/google/cloud/tools/jib/docker/DockerClientTest.java

```diff
@@ -176,11 +181,37 @@ public void testTag() throws InterruptedException, IOException, InvalidImageRefe
               Assert.assertEquals(Arrays.asList("tag", "original", "new"), subcommand);
               return mockProcessBuilder;
             });
-    Mockito.when(mockProcess.waitFor()).thenReturn(0);
+    when(mockProcess.waitFor()).thenReturn(0);
 
     testDockerClient.tag(ImageReference.of(null, "original", null), ImageReference.parse("new"));
   }
 
+  @Test
+  public void testDefaultClient() {
+    DockerClient dc = DockerClient.newClient();
+    Assert.assertEquals(
+        dc.getProcessBuilderFactory().apply(Collections.emptyList()).command(),
+        Collections.singletonList("docker"));
+    Assert.assertEquals(
+        dc.getProcessBuilderFactory().apply(Collections.emptyList()).environment(),
+        System.getenv());
+  }
+
+  @Test
+  public void testCustomClient() {
+    Path path = Paths.get("/path.to/docker");
+    Map<String, String> environment = new HashMap<String, String>();
+    environment.putAll(System.getenv());
+    environment.putAll(Collections.singletonMap("Key1", "Value1"));
+
+    DockerClient dc = DockerClient.newClient(path, environment);
+    Assert.assertEquals(
+        dc.getProcessBuilderFactory().apply(Collections.emptyList()).command(),
```

Comment:
nit: the `dc` name, and the order of assert arguments.

Reply:
addressed